### PR TITLE
HRIS-127 [Markup] Create Add Employee Modal

### PR DESCRIPTION
**Pull request by JohnRoyCabezas**
## Issue Link

[Backlog Link](https://framgiaph.backlog.com/board/HRIS?selectedIssueKey=HRIS-127&category=342155&assignee=405757)

## Definition of Done

- [x] Users can click the Add New Employee button, the Add New Employe Modal shall popup and allow the user to create a new employee.

## Pre-condition

npm run dev 

or

docker compose up

## Expected Output

Upon clicking the Add New Employee button, the Add New Employe Modal shall popup and allow the user to create a new employee.

## Screenshots/Recordings

Employee Modal (Desktop)
image

Employee Modal Input (Desktop)
image

Employee Modal Validation (Desktop)
image

Employee Modal (Mobile)
image

image

### CODE REVIEW

File: client/src/components/molecules/AddNewEmployeeModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    })
+  }
+
+  return (
+    <ModalTemplate
+      {...{
+        isOpen,
+        closeModal
+      }}
+      className="w-full max-w-[700px]"
```
Comment (jsvelte) [inline client/src/components/molecules/AddNewEmployeeModal/index.tsx:76]:
```suggestion
      className="w-full max-w-[430px]"
```

**Review by jsvelte — APPROVED** [review]
# Approved 💯

File: client/src/components/molecules/AddNewEmployeeModal/index.tsx (hunk 1 above)
Comment (jsvelte) [reply client/src/components/molecules/AddNewEmployeeModal/index.tsx:76]: Please look at the size in the figma
image

File: client/src/components/molecules/AddNewEmployeeModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+              />
+            </TextField>
+            {errors?.last_name !== null && errors?.last_name !== undefined && (
+              <span className="error text-[10px]">{errors.last_name?.message}</span>
+            )}
+          </section>
+        </main>
+
+        {/* Custom Modal Footer Style */}
+        <ModalFooter>
```
Comment (jsvelte) [inline client/src/components/molecules/AddNewEmployeeModal/index.tsx:184]: Add also `reset` and `cancel` button here

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**